### PR TITLE
v7.5.0

### PR DESCRIPTION
**Pull request by Pytal**
## [v7.5.0](https://github.com/nextcloud/nextcloud-vue/tree/v7.5.0) (2023-01-19)

[Full Changelog](https://github.com/nextcloud/nextcloud-vue/compare/v7.4.0...v7.5.0)

### :rocket: Enhancements

- Set placement of select dropdown [\#3656](https://github.com/nextcloud/nextcloud-vue/pull/3656) ([Pytal](https://github.com/Pytal))
- Breadcrumb: add exact prop and fix style [\#3654](https://github.com/nextcloud/nextcloud-vue/pull/3654) ([skjnldsv](https://github.com/skjnldsv))

### CODE REVIEW

**Comment by skjnldsv** [comment]
Please put the changelog in the first post

**Comment by raimund-schluessler** [comment]
Can we rebase so we get #3665 in as well?

**Comment by PVince81** [comment]
@raimund-schluessler if urgent, can you do this yourself and update the changelog commit + top post ?

**Comment by skjnldsv** [comment]
It should not be in the changelog as it's documentation.
Changelog = library related changes only :)

**Comment by skjnldsv** [comment]
Btw there is a mistake in the changelog, it seems I opened a duplicate PR with 
- https://github.com/nextcloud/nextcloud-vue/pull/3654
- https://github.com/nextcloud/nextcloud-vue/pull/3653

https://github.com/nextcloud/nextcloud-vue/pull/3654 got reviewed and merged, so github assumed https://github.com/nextcloud/nextcloud-vue/pull/3653 was too since the commit was the same.
We can remove https://github.com/nextcloud/nextcloud-vue/pull/3653 from the changelog :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 
+## [v7.5.0](https://github.com/nextcloud/nextcloud-vue/tree/v7.5.0) (2023-01-18)
+
+[Full Changelog](https://github.com/nextcloud/nextcloud-vue/compare/v7.4.0...v7.5.0)
+
+### :rocket: Enhancements
+
+- Set placement of select dropdown [\#3656](https://github.com/nextcloud/nextcloud-vue/pull/3656) ([Pytal](https://github.com/Pytal))
+- Breadcrumb: add exact prop and fix style [\#3654](https://github.com/nextcloud/nextcloud-vue/pull/3654) ([skjnldsv](https://github.com/skjnldsv))
+- Breadcrumb: add exact prop and fix style [\#3653](https://github.com/nextcloud/nextcloud-vue/pull/3653) ([skjnldsv](https://github.com/skjnldsv))
```
Comment (skjnldsv) [inline CHANGELOG.md:13]:
```suggestion
- Breadcrumb: add exact prop and fix style [\#3654](https://github.com/nextcloud/nextcloud-vue/pull/3654) ([skjnldsv](https://github.com/skjnldsv))
```

**Review by skjnldsv — CHANGES_REQUESTED** [review]

**Comment by raimund-schluessler** [comment]
> It should not be in the changelog as it's documentation.
> 
> Changelog = library related changes only :)

Nevermind, I thought it's a bugfix. If it's only documentation, it doesn't need to be in the release.

**Review by raimund-schluessler — APPROVED** [review]

**Comment by Pytal** [comment]
Able to release as admin @skjnldsv? https://github.com/nextcloud/nextcloud-vue/actions/runs/3963753999

**Comment by Pytal** [comment]
👆 Please ignore @skjnldsv :) Already done by @AndyScherzinger :)

**Pull request merged**